Makefile

```diff
@@ -384,7 +391,7 @@ launch-logging: check-kubectl check-kubectl-connection
 	kubectl --namespace=monitoring port-forward `kubectl --namespace=monitoring get pods -l k8s-app=kibana-logging -o json | jq '.items[0].metadata.name' -r` 35601:5601 &
 
 launch-loki:
-	helm repo add --force-update loki https://grafana.github.io/loki/charts
+	helm repo add loki https://grafana.github.io/loki/charts
```

Comment:
Do newer versions of help have a replacement for this flag? The reason we added it is that retries in Travis break if it's not there. `travis_retry` runs the tests for a second time, which includes running this command, and then it fails twice with `repo "loki" already exists`. Maybe adding something like `helm repo delete loki || true` beforehand would work?

Reply:
They don't. I'll do as you suggest. Thanks!

Reply:
You can also pre-load this file, have a look at `cat ~/.helm/repository/repositories.yaml`

Reply:
I went with Matthew's suggestion for expediency.